src/components/shared/Entity/Details/History.tsx

```diff
@@ -0,0 +1,123 @@
+import { Box, CircularProgress, Grid, List, ListItem } from '@mui/material';
+import { PublicationSpecsExt_PublicationHistory } from 'api/publicationSpecsExt';
+import KeyValueList from 'components/shared/KeyValueList';
+import Tile from 'components/shared/Tile';
+import { addDays, endOfMonth, format, startOfMonth, subDays } from 'date-fns';
+import useGlobalSearchParams, {
+    GlobalSearchParams,
+} from 'hooks/searchParams/useGlobalSearchParams';
+import usePublicationSpecsExt_History from 'hooks/usePublicationSpecsExt';
+import { useMemo } from 'react';
+import { useIntl } from 'react-intl';
+
+const CARD_DATE_FORMAT = 'EEEE, MMM do, yyyy';
+const CALENDAR_DATE_FORMAT = 'yyyy-MM-dd';
+
+export const convertData = (
```

Comment:
No - I want a general name on purpose. The idea here is that we'll need to consume our data and then output it in a manner that makes it useful for eCharts. This would need to keep growing slowly and I do not want to have to keep updating it.

Potentially as it grows we can make other functions with specific names.